src/Tempest/Console/src/Discovery/ConsoleCommandDiscovery.php

```diff
@@ -41,7 +42,7 @@ public function createCachePayload(): string
 
     public function restoreCachePayload(Container $container, string $payload): void
     {
-        $commands = unserialize($payload, ['allowed_classes' => [ConsoleCommand::class, MethodReflector::class]]);
+        $commands = unserialize($payload, ['allowed_classes' => [GeneratorCommand::class, ConsoleCommand::class, MethodReflector::class]]);
```

Comment:
This is one of the reasons I'm hesitant about "a special kind of command class": we need several changes all over the place to make sure they work. That leads to a brittle and complicated design.